From: William Grzybowski <william@grzy.org>
Date: Thu, 18 Jul 2019 22:10:18 -0300
Subject: Do not install custom.js for codefund.app because of privacy breach

Forwarded: not-needed
---
 docs/conf.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index dbe0d6e..555b21a 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -324,8 +324,3 @@ autosectionlabel_prefix_document = True
 # http://www.sphinx-doc.org/en/master/usage/extensions/doctest.html
 doctest_global_setup = """
 """
-
-
-def setup(app):
-    """https://docs.readthedocs.io/en/latest/guides/adding-custom-css.html#adding-custom-css-or-javascript-to-a-sphinx-project"""
-    app.add_js_file('custom.js')
