From: =?utf-8?q?H=C3=A5vard_Flaget_Aasen?= <haavard_aasen@yahoo.no>
Date: Sun, 27 Dec 2020 18:39:08 +0100
Subject: drop intersphinx

Forwarded: not-needed
---
 docs/conf.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 5709486..4237817 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -45,7 +45,6 @@ extensions = [
     # 'sphinx.ext.coverage',
     'sphinx.ext.doctest',
     'sphinx.ext.extlinks',
-    'sphinx.ext.intersphinx',
     'sphinx.ext.viewcode',
     # 'sphinx.ext.todo',
 ]
@@ -56,7 +55,6 @@ except ImportError:
 else:
     extensions.append('sphinx_copybutton')
 
-intersphinx_mapping = {'python': ('https://docs.python.org/3', None)}
 #todo_include_todos = True
 
 # Add any paths that contain templates here, relative to this directory.
